package-lock.json

```diff
@@ -1,8 +1,8151 @@
 {
   "name": "azure-rest-api-specs-tests",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Please revert changes to package-lock.json

Reply:
done